uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-entitlements.html

```diff
@@ -1 +1,154 @@
-Entitlements
+<section class="panel panel-default panel-default-plain">
+  <header class="panel-heading">
+    <h2 class="panel-title">Contract Entitlements</h2>
+    <hr/>
```

Comment:
The HR tag adds a margin to the panel heading. It's style is defined by bootstrap as .panel-heading>hr. This markup is also used in other places, such as Contact Summary tab.